drivers/SmartThings/jbl/src/jbl/capability_handler.lua

```diff
@@ -0,0 +1,138 @@
+local log = require "log"
+local json = require "st.json"
+local utils = require "st.utils"
+local fields = require "fields"
+
+local capabilities = require "st.capabilities"
+
+local capability_handler = {}
+capability_handler.__index = capability_handler
+
+local function smartthings_playback_capability_handler(driver, device, capability_status)
+    local st_status_to_jbl_playback_status_table = {
+        paused = "pause",
+        playing = "play",
+    }
+
+    local conn_info = device:get_field(fields.CONN_INFO)
+    log.info(string.format("media-playback.set_playback_status_handler : dni = %s, status = %s", device.device_network_id, capability_status))
+
+    local jbl_playback_status = st_status_to_jbl_playback_status_table[capability_status]
+
+    local response, err, status = conn_info:post_playback(string.format('{"playback": "%s"}', jbl_playback_status))
+    if not err and status == 200 then
+        device:emit_event(capabilities.mediaPlayback.playbackStatus[capability_status]())
```

Comment:
By convention for LAN devices, we try to avoid doing a "synthetic" emit on success for stuff like this. The appropriate design pattern would be to send the command, and then wait for the appropriate state change to come in on the server sent event stream asynchronously.

Reply:
device:emit_event removed